tests/test_nepc.py

```diff
@@ -84,6 +84,16 @@ def test_CS_class(nepc_connect):
     assert isinstance(cs.data["sigma"], list)
     assert isinstance(cs.data["sigma"][0], float)
     # test len()
+    assert(len(cs) == len(cs.data['e']))
+    # test str()
+    undef = "\\N"
+    message = f"cross section id: {cs.metadata['cs_id']}\n" \
```

Comment:
Same comment as above. I think we want to use hard-wired values to test against.